src/components/experiment-tracking/time-series/time-series.js

```diff
@@ -75,6 +76,13 @@ export const TimeSeries = ({ metricsData, selectedRuns }) => {
 
   const xScale = d3.scaleTime().domain([minDate, maxDate]).range([0, width]);
 
+  // if (currentZoomState) {
+  //   xScale.domain(currentZoomState);
+  // }
```

Comment:
Is this still needed? Can remove it if it's not.

Reply:
Yes, this will be removed.